### DISCUSSION TITLE
In-depth tutorials?

### DISCUSSION THREAD

**Discussion by Woisek** [rsxdalv/TTS-WebUI#187] [Q&A]
This whole project looks interesting but it lacks of explanations.
I would love to see or read tutorials on the specific parts of the webui. Especially voice cloning. I followed the three videos but I couldn't manage to get a 'cloned' voice. It generates something, but it's far from the source, aka 'cloning'.
So it would be very helpful to show the steps need for that, with an explanation of the parameters to set to get an acceptable cloned voice. Although it's funny to hear me in different variants, it's not the purpose of this tool, I guess.

**Comment by rsxdalv** [comment]
Hopefully there will be more in the way of tutorials. Also I am slowly but steadily working on a non-gradio UI that will have better UX to begin with, enabling more info to be right on the page where you work from. 

Also if you see any helpful videos or articles I can always add links in the README to help people understand this project. It was meant to be very simple but the ability to make simple things with AI and Gradio slowly went to 0.

**Comment by AIhasArrived** [comment]
Hello to both, did you get /find any new guides and stuff?

**Reply by rsxdalv** [reply]
I'm building the React UI nowadays

**Reply by AIhasArrived** [reply]
Since I have you here, I have 2 questions:
- Are there others places where the community speak about and around this ui out there? 
- I just watched a tuttorial and got into your UI, for the cloning part it seems it only needs a clip of 20 seconds right? My question is is it possible to "inject" more clips, many more, to ensure it generates a better and better voice please? Does that technology exists? Before coming here I was testing dl-art-school and did not got good results despite being able to send looot of "input voices" the problem was it took so much time to genrate LOT of models, then you have to test them all.. ANYWAY, i would like more options of the cloning, adding options where we can "insert more = wait for a longer time process" but in exchange ensure somehow obtaining a better reuslt in term of voice cloning, what do you thing?

**Reply by rsxdalv** [reply]
> Since I have you here, I have 2 questions:
> 
> * Are there others places where the community speak about and around this ui out there?
> * I just watched a tuttorial and got into your UI, for the cloning part it seems it only needs a clip of 20 seconds right? My question is is it possible to "inject" more clips, many more, to ensure it generates a better and better voice please? Does that technology exists? Before coming here I was testing dl-art-school and did not got good results despite being able to send looot of "input voices" the problem was it took so much time to genrate LOT of models, then you have to test them all.. ANYWAY, i would like more options of the cloning, adding options where we can "insert more = wait for a longer time process" but in exchange ensure somehow obtaining a better reuslt in term of voice cloning, what do you thing?

As for the cloning, I am actually not fully dedicated to making the best possible clones since that is something that would increase the risk to this technology. I think youtube has videos about Tortoise and RVC training, I think that's close to the state of the art for open source. Bark _might_ be able to do better; however, training for bark is not fully available so it will limit the quality of the results quite a lot. 
As for the other places, I am not sure about all of the places, I mentioned some in the other comment.
